examples/prompt/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"github.com/bitfield/script"
+)
+
+func main() {
+	script.Prompt("This is an example, what do you think? ", "It's awesome").Stdout()
```

Comment:
Yes, it's awesome, but I have a rule of thumb for example programs in this repo which says that they should not just demonstrate the feature, but actually show why it's useful. Why is it useful to be able to get input from the user? What real-world problem might we be solving by doing this? Let's write _that_ program.

This is helpful not only because it gives users something they can copy and modify for their own programs, but it also forces us to do some thinking. _Is_ the feature really useful, and if so, for what? That helps us design the feature.

Reply:
I have updated the example with our installation example discussed on the issue #51